Dockerfile

```diff
@@ -1,29 +1,115 @@
-FROM ruby:2.7.2
-
-# 1. Configuring apt to properly install yarn
-# 2. Installing node, postgres-client, yarn
-# 3. Clearing apt-cached files to tighten the image size
-RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | APT_KEY_DONT_WARN_ON_DANGEROUS_USAGE=1 apt-key add - \
-  && echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list \
-  && apt-get update -qq \
-  && apt-get install -y build-essential \
-  nodejs \
-  yarn \
-  && rm -rf /var/lib/apt/lists/*
+# Generating a generic Rails image.
+# Just for caching.
+FROM ruby:2.7-alpine AS rails
 
-WORKDIR /app
+# `apk` is like `apt`, but for Alpine Linux.
+RUN apk add --update --no-cache --quiet \
+    build-base \
+    nodejs-current \
+    postgresql-dev \
+    sqlite-dev \
```

Comment:
BEGONE SQLITE! YOU ARE NOT WELCOME HERE!

Reply:
Should be gone now.  👍